frontend/src/i18n/it/index.ts

```diff
@@ -0,0 +1,396 @@
+        },
+      },
+      create: {
+        pageTitle: 'Crea il mio Bot',
+      },
+      edit: {
+        pageTitle: 'Modifica il mio Bot',
+      },
+
+      item: {
+        title: 'Nome',
+        description: 'Descrizione',
+        instruction: 'Istruzioni',
+      },
+      apiSettings: {
+        pageTitle: 'Configurazioni API Pubbliche del Bot Condiviso',
+        label: {
+          endpoint: 'Endpoint API',
+          usagePlan: 'Piano di utilizzo',
+          allowOrigins: 'Origini consentite',
+          apiKeys: 'Chiavi API',
+          period: {
+            day: 'Al GIORNO',
+            week: 'Alla SETTIMANA',
+            month: 'Al MESE',
+          },
+          apiKeyDetail: {
+            creationDate: 'Data di creazione',
+            active: 'Attino',
+            inactive: 'Inattivo',
+            key: 'Chiavi API',
+          },
+        },
+        item: {
+          throttling: 'Limitazione',
+          burstLimit: 'Velocità massima di trasferimento',
+          rateLimit: 'Richieste al secondo',
+          quota: 'Quota',
+          requestLimit: 'Richieste',
+          offset: 'Slittamento',
+        },
+        help: {
+          overview:
+            "La creazione di un'API consente l'accesso alle funzioni del Bot da parte di client e applicazioni esterni.",
+          endpoint: 'Il client può utilizzare il Bot da questo endpoint.',
+          usagePlan:
+            "I piani di utilizzo specificano il numero o la frequenza di richieste che la tua API accetta da un client. Associa un'API a un piano di utilizzo per tenere traccia delle richieste ricevute dalla tua API.",
+          throttling: 'Limita la frequenza con cui gli utenti possono chiamare la tua API.',
+          rateLimit:
+            'Inserisci la velocità, in richieste al secondo, con cui i client possono chiamare la tua API.',
+          burstLimit:
+            'Inserisci il numero di richieste simultanee che un client può effettuare alla tua API.',
+          quota:
+            'Attiva le quote per limitare il numero di richieste che un utente può effettuare alla tua API in un determinato periodo di tempo.',
+          requestLimit:
+            "Inserisci il numero totale di richieste che un utente può effettuare nel periodo di tempo selezionato nell'elenco a discesa.",
+          allowOrigins:
+            "Origini client consentite per l'accesso. Se l'origine non è consentita, il chiamante riceve una risposta 403 Forbidden e gli viene negato l'accesso all'API.""L'origine deve seguire il formato:(http|https)://nome-host o (http|https)://nome-host:porta e possono essere utilizzati i caratteri jolly(*).",
```

Comment:
Still have lint error:  
https://github.com/aws-samples/bedrock-claude-chat/actions/runs/8846261319/job/24339193294?pr=282

Could you check it again? I recommend you to run following commands before commit to make sure the lint issue  is solved.

```
cd frontend
npm ci
npm run lint
```